spec/acceptance/1_default_parameters_spec.rb

```diff
@@ -23,8 +23,8 @@
         cleanup         => true,
       } ->
       class { 'confluence':
-        version      => '5.5.6',
-        jvm_type     => 'oracle-jdk-1.8',
+        version      => '8.5.12',
+        jvm_type     => 'openjdk-11',
```

Comment:
Seems like my change to the archive resource above in 2a4bfcc911f8c19961e9af6638a684f0e971e6f6 is incomplete ...
Title 'jdk11' but it downloads zulu jdk8.